Services/CO.CDP.Tenant.WebApi/Program.cs

```diff
@@ -15,10 +15,16 @@
 var builder = WebApplication.CreateBuilder(args);
 builder.ConfigureForwardedHeaders();
 
+var enableSwagger = builder.Configuration.GetSection("Features").GetValue<bool>("SwaggerUI");
```

Comment:
Use the `builder.Configuration.GetValue("Features:SwagerUI", false)` syntax please, as it won’t throw an exception if the section isn’t defined but will fallback to the default value instead.